pkg/ccl/backupccl/show.go

```diff
@@ -926,6 +940,42 @@ func nullIfZero(i descpb.ID) tree.Datum {
 	return tree.NewDInt(tree.DInt(i))
 }
 
+func showRegions(descriptor catalog.TypeDescriptor, dbname string) string {
+	var regionsStringBuilder strings.Builder
+	if descriptor == nil {
+		return ""
+	}
+
+	primaryRegionName, err := descriptor.PrimaryRegionName()
+	if err != nil {
+		return ""
+	}
+	regionsStringBuilder.WriteString("ALTER DATABASE ")
+	regionsStringBuilder.WriteString(dbname)
+	regionsStringBuilder.WriteString(" PRIMARY REGION ")
+	regionsStringBuilder.WriteString("\"" + primaryRegionName.String() + "\"")
+	regionsStringBuilder.WriteString("; ")
+
+	regionNames, err := descriptor.RegionNames()
+	if err != nil {
+		return ""
+	}
+	for i, regionName := range regionNames {
+		if regionName != primaryRegionName {
+			regionsStringBuilder.WriteString("ALTER DATABASE ")
+			regionsStringBuilder.WriteString(dbname)
+			regionsStringBuilder.WriteString(" ADD REGION ")
+			regionsStringBuilder.WriteString("\"" + regionName.String() + "\"")
+			if i == len(regionNames)-1 {
```

Comment:
nit: remove this special casing , and always add ";", and refactor to " ALTER DATABASE "... i.e. there's a space before the A in ALTER.